### PR TITLE
Added camera movement script

### PR DESCRIPTION
**Pull request by mariaFernando**
Zooms with mouse wheel, pans with mouse right button

### CODE REVIEW

File: UnityProject/Assets/Scripts/Camera/MoveCamera.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+using UnityEngine;
+using System.Collections;
+
+public class MoveCamera : MonoBehaviour 
+{
+	public float panSpeed = 1.0f;
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: Indentation out of place everywhere in this file.

File: UnityProject/Assets/Scripts/Camera/MoveCamera.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        // Mouse released
+        if (Input.GetMouseButtonUp(1)) 
+            isPanning = false;
+
+		// Move the camera on its XY plane
+		if (isPanning){
+            Vector3 currPos = Camera.main.ScreenToViewportPoint(Input.mousePosition - firstPos); // Current mouse position
+
+            // If the mouse hasn't moved, the pan stops
+            // if (currPos != lastPos)
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: Remove commented code

File: UnityProject/Assets/Scripts/Camera/MoveCamera.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+               // lastPos = currPos;
+            // }
+		}
+
+        // ZOOM
+        // Mouse scrollwheel to zoom in fixed increments
+        float scroll = Input.GetAxis("Mouse ScrollWheel");
+        if (scroll > 0)
+            Camera.main.orthographicSize -= zoomSpeed;
+        else if (scroll < 0)
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: Could this just be an `else`?

Comment (mariaFernando) [reply UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: If scroll == 0 the camera's orthographic size doesn't change

File: UnityProject/Assets/Scripts/Camera/MoveCamera.cs (hunk 1 above)
Comment (mariaFernando) [reply UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: Riiiight I'm on it, sorry ^^"

File: UnityProject/Assets/Scripts/Camera/MoveCamera.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        // ZOOM
+        // Mouse scrollwheel to zoom in fixed increments
+        float scroll = Input.GetAxis("Mouse ScrollWheel");
+
+        if (scroll > 0)
+            Camera.main.orthographicSize -= zoomSpeed;
+        else if (scroll < 0)
+            Camera.main.orthographicSize += zoomSpeed;
+    }
+}
```
Comment (OlafSzmidt) [inline UnityProject/Assets/Scripts/Camera/MoveCamera.cs]: EOF line missing, github does go crazy sometimes so dont worry if you cant fix this though

**Pull request merged**